dd-sdk-android-internal/src/main/java/com/datadog/android/internal/telemetry/UploadQualityEvent.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.internal.telemetry
+
+/**
+ * The different categories of upload quality events.
+ * - COUNT: The number of uploads.
+ * - BLOCKER: The count of different blockers that prevented an upload.
+ * - FAILURE: The count of different failure codes.
```

Comment:
minor: these should probably go to the individual values doc instead of enum doc. Same for enum below.